add-on/chrome/bootstrap.js

```diff
@@ -646,6 +654,31 @@ var WindowListener = {
           }
       },
 
+      handleMousemove: function(event) {
+        // Only update every so often.
+        let now = Date.now();
+        if (now - this.lastCursorTime < MIN_CURSOR_INTERVAL) {
+          return;
+        }
+        this.lastCursorTime = now;
+
+        // Skip the update if cursor is out of bounds or didn't move much.
+        let browserBox = gBrowser.selectedBrowser.boxObject;
+        let deltaX = event.screenX - browserBox.screenX;
+        let deltaY = event.screenY - browserBox.screenY;
+        if (deltaX < 0 || deltaX > browserBox.width ||
+            deltaY < 0 || deltaY > browserBox.height ||
```

Comment:
As commented elsewhere about percentages, I think it would be safer to divide by `browserBox.width` instead of `window.outerWidth` as we're already using it to compute bounds.